src/MotorTransport.java

```diff
@@ -0,0 +1,8 @@
+public abstract class MotorTransport {
```

Comment:
Нужно расширить класс Transport, так как мы хотим унаследовать методы и поля из Transport
```suggestion
public abstract class MotorTransport extends Transport{
```

Reply:
И нужно будет добавить конструктор

Reply:
Также в этом классе мы переопределим метод check()
Мы вызовем родительский метод check() и добавим проверку двишка:
```
    @Override
    public void check() {
        super.check();
        //вызвать метод checkEngine()
    }

```